docs/deployment/composer.md

```diff
@@ -1,5 +1,7 @@
 Drupal VM is configured to use `composer create-project` to build a Drupal 8 codebase by default but supports building Drupal from a custom `composer.json` file as well.
 
+_Note that if you already have a project built using a `composer.json` file you should instead add [Drupal VM as a dependency](composer-dependency.md) of your project. The method described below will make it somewhat difficult to manage your `composer.json` together with Drupal VM as the file will be copied from `drupal_composer_path` into `drupal_composer_install_dir` as a secondary `composer.json`._
```

Comment:
Or well that's a bit odd too. Maybe it should be renamed to `drupal_composer_copy_path` so it makes more sense as to why you need to set it to `false` for the composer dependency setup.